src/ui/scss/component/_comments.scss

```diff
@@ -0,0 +1,67 @@
+.comment {
+  padding-bottom: var(--spacing-vertical-large);
+
+  &.reply {
+    border-left: 3px solid var(--lbry-teal-5);
+    padding-left: var(--spacing-vertical-large);
+    position: relative;
+    top: -1px;
+  }
+
+  &:not(:first-of-type) {
+    padding-top: var(--spacing-vertical-large);
+  }
+
+  &:not(:last-of-type) {
+    border-bottom: 1px solid var(--lbry-gray-1);
+
+    [data-mode='dark'] & {
+      border-color: rgba($lbry-gray-5, 0.2);
+    }
+  }
+}
+
+.comment__actions-wrap {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  width: 4.5rem;
+}
+
+.comment__action {
+  @include hide-text;
+  width: 0; height: 0;
+  transition: border-color 0.2s;
+
+  &.downvote {
```

Comment:
We're not doing upvotes/downvotes. The only time I can see that is if when move to using claims as comments, we can count supports with say, 10 confirmations as being upvotes.